### ISSUE TITLE
exit code 1 when panic are found?

### ISSUE THREAD

**Issue by dotansimha** [YassinEldeeb/rust-panic-analyzer#2]
I think in order to have a better integration with CIs, we can use exit code != 0 to indicate a non-expected panics?

**Comment by YassinEldeeb** [comment]
yeah, I think it’s a good idea. but this is gonna be extreme for already existing projects with lots of panic points, they wouldn’t be able to merge anything unless they take the time to fix all the issues which takes a lot of time and effort.

maybe we can add an optional non default `STRICTLY_FAIL` environment variable for strictly failing if panic points were detected instead of just acknowledging them?

what do you think? @dotansimha 🤔
